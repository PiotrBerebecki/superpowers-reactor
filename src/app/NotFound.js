import React from 'react';
import { ErrorCard } from './../styled';
import Main from './Main';

function NotFound() {
  return (
    <Main>
      <ErrorCard>
        Sorry, this page has not been found.
      </ErrorCard>
    </Main>
  );
}

export default NotFound;
