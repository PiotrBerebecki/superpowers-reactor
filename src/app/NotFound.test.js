/**
 * @jest-environment node
 */

import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import { MemoryRouter } from 'react-router-dom';
import 'jest-styled-components';

import NotFound from './NotFound';

describe('NotFound', () => {
  it('renders', () => {
    const wrapper = shallow(<NotFound />);
    expect(shallowToJson(wrapper)).toMatchStyledComponentsSnapshot();
  });
});
