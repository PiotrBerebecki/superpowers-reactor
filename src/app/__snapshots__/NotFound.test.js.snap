// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound renders 1`] = `
<Main>
  <styled.div>
    Sorry, this page has not been found.
  </styled.div>
</Main>
`;
